### PR TITLE
Security now always get deathrattle implants

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Security now always get deathrattle implants. Note that only security can hear their own deathrattles. Also note that this doesn't roll if the entire station has deathrattle implants.

Mechanically, this just forces the deathrattle security event.

## Why It's Good For The Game

Adds an extra layer of protection for security officers killed in the line of duty. Penalizes random green-shift assassinations of security players just because they're security. This should not affect traitors normally/"fairly" killing security considering:
- Security constantly broadcasts their location anyways.
- Suit sensors exist, and a paramedic/warden/AI usually snitches.
- An AI usually watches most conflicts.

## Proof Of Testing

image

## Changelog

:cl: BurgerBB
balance: Security now always get deathrattle implants. Note that only security can hear their own deathrattles.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/security_deathrattle/security_deathrattle.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/datum/station_trait/deathrattle_department/security
+	weight = 0
+
+/datum/controller/subsystem/processing/station/SetupTraits()
+
+	. = ..()
+
+	var/datum/station_trait/deathrattle_department/security/existing_security_trait = locate() in station_traits
+	if(existing_security_trait)
+		return .
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/security_deathrattle/security_deathrattle.dm:12]: Is this really needed? With the weight set to 0 this should never prop, no?

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_zubbers/code/modules/security_deathrattle/security_deathrattle.dm (hunk 1 above)
Comment (BurgerLUA) [reply modular_zubbers/code/modules/security_deathrattle/security_deathrattle.dm:12]: true, but it's just in case:tm:

**Comment by BurgerLUA** [comment]
> Maybe a research item that can be disabled if comms get disabled? Could put it under tracking tech but a full blown round start deathrattle implant is kind of a lot. Especially since death rattle announces go the entire station.

Deathrattle implants only go to people with a deathrattle implant.

**Comment by The-Sun-In-Splendour** [comment]
Can we EMP deathrattles? This feels like a significant buff to security?

**Pull request closed without merging**